configure.ac

```diff
@@ -284,6 +286,9 @@ AC_MSG_CHECKING([what semaphore type to use]);
 AS_IF([test "x$have_mach" = "xtrue"],
   [AC_DEFINE(USE_MACH_SEM, 1, [Define to use Mach semaphores])
     AC_MSG_RESULT([Mach semaphores])],
+  [test "x$have_futex" = "xtrue"],
+  [AC_DEFINE(USE_FUTEX_SEM, 1, [Define to use Futex semaphores])
```

Comment:
In this particular case we want to be symmetric on the USE_xxxxx_SEM, so that should stay.
Will see whether we can be more generic here from which we derive USE_FUTEX_SEM.